modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -100,6 +99,9 @@
 	public static final String BUILD_CLIENT_EXTENSION_ZIP_TASK_NAME =
 		"buildClientExtensionZip";
 
+	public static final String CLIENT_EXTENSION_BUILD_DIR =
```

Comment:
Their may be a more idiomatic way to do this in gradle, and also would allow this to automatically be configurable.  Maybe we can ping Peter about it.  But otherwise this looks good.